bitnami/opensearch/templates/master/statefulset.yaml

```diff
@@ -107,20 +106,16 @@ spec:
           resources: {{- include "common.resources.preset" (dict "type" .Values.volumePermissions.resourcesPreset) | nindent 12 }}
           {{- end }}
           volumeMounts:
-            - name: empty-dir
-              mountPath: /tmp
-              subPath: tmp-dir
```

Comment:
Why are you removing this?

Reply:
The init-container isn't using the `/tmp` folder